common/app/views/fragments/inlineJSNonBlocking.scala.html

```diff
@@ -30,4 +30,32 @@
 // Ophan pageview ID and browser ID are needed by Google Analytics, which runs just after this script tag
 @InlineJs(ophanConfig().body, "ophanConfig.js")
 
+@if(WebpackSwitch.isSwitchedOn) {
```

Comment:
Reviewing this suggestion, I have decided to remove the Webpack-specific version of `standard/main.js` and add a switch check to the original `standard/main.js` to prevent us from loading interactives twice.